counterpartylib/lib/messages/issuance.py

```diff
@@ -273,10 +340,17 @@ def parse (db, tx, message):
                 'asset_id': str(asset_id),
                 'asset_name': str(asset),
                 'block_index': tx['block_index'],
+                'asset_longname': subasset_longname,
             }
-            sql='insert into assets values(:asset_id, :asset_name, :block_index)'
+            sql='insert into assets values(:asset_id, :asset_name, :block_index, :asset_longname)'
             issuance_parse_cursor.execute(sql, bindings)
 
+    if reissuance:
```

Comment:
1. Good find.  Changed to `if status == 'valid' and reissuance:` 
2. This is also related to the above discussion.  We can remove it if we also get rid of `reissued_asset_longname`.